2019-10-13T12:09:55,419 [ERROR] (DBUserDAO.java:63) - java.sql.SQLSyntaxErrorException: Table 'hospital2.specialization' doesn't exist
2019-10-13T12:18:03,548 [ERROR] (DBUserDAO.java:63) - java.sql.SQLSyntaxErrorException: Table 'hospital2.specialization' doesn't exist
2019-10-13T12:18:28,999 [ERROR] (DBUserDAO.java:63) - java.sql.SQLSyntaxErrorException: Table 'hospital2.specialization' doesn't exist
2019-10-13T12:19:01,615 [ERROR] (DBUserDAO.java:63) - java.sql.SQLSyntaxErrorException: Table 'hospitaltest.specialization' doesn't exist
2019-10-13T12:20:11,536 [ERROR] (DBUserDAO.java:63) - java.sql.SQLSyntaxErrorException: Table 'hospitaltest.specialization' doesn't exist
2019-10-13T15:27:33,289 [ERROR] (DBUserDAO.java:111) - java.sql.SQLException: Field 'iduser' doesn't have a default value
2019-10-13T16:41:33,118 [ERROR] (ConnectionPool.java:77) - java.sql.SQLException: No timezone mapping entry for 'UTC   '
2019-10-13T16:41:33,137 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-13T16:44:55,047 [ERROR] (ConnectionPool.java:77) - java.sql.SQLException: No timezone mapping entry for 'UTC   '
2019-10-13T16:44:55,066 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-13T16:47:13,366 [ERROR] (ConnectionPool.java:77) - java.sql.SQLException: No suitable driver found for THIS LINE GENERATED JUNIT TEST
2019-10-13T16:47:13,382 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-13T16:52:19,675 [ERROR] (ConnectionPool.java:77) - java.sql.SQLException: No suitable driver found for THIS LINE GENERATED JUNIT TEST
2019-10-13T16:52:19,692 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-13T16:57:53,287 [ERROR] (ConnectionPool.java:77) - java.sql.SQLException: No suitable driver found for THIS LINE GENERATED JUNIT TEST
2019-10-13T16:57:53,304 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-13T17:00:00,912 [ERROR] (ConnectionPool.java:77) - java.sql.SQLException: No suitable driver found for THIS LINE GENERATED JUNIT TEST
2019-10-13T17:00:00,929 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-13T17:00:28,682 [ERROR] (ConnectionPool.java:77) - java.sql.SQLException: No suitable driver found for THIS LINE GENERATED JUNIT TEST
2019-10-13T17:00:28,698 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-13T17:05:46,609 [ERROR] (ConnectionPool.java:77) - java.sql.SQLException: No suitable driver found for THIS LINE GENERATED JUNIT TEST
2019-10-13T17:05:46,624 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-13T17:07:52,413 [ERROR] (ConnectionPool.java:77) - java.sql.SQLException: No suitable driver found for THIS LINE GENERATED JUNIT TEST
2019-10-13T17:07:52,430 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-13T17:08:24,700 [ERROR] (ConnectionPool.java:77) - java.sql.SQLException: No timezone mapping entry for 'UTC   '
2019-10-13T17:08:24,718 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-13T17:56:12,422 [ERROR] (DBUserDAO.java:271) - java.sql.SQLSyntaxErrorException: Table 'test_jd2.patient' doesn't exist
2019-10-13T17:56:12,446 [ERROR] (DBUserDAO.java:232) - java.lang.NullPointerException
2019-10-13T18:06:01,038 [ERROR] (DBUserDAO.java:271) - java.sql.SQLSyntaxErrorException: Table 'test_jd2.patient' doesn't exist
2019-10-13T18:06:01,064 [ERROR] (DBUserDAO.java:232) - java.lang.NullPointerException
2019-10-14T01:13:34,586 [ERROR] (ConnectionPool.java:77) - java.sql.SQLNonTransientConnectionException: Data source rejected establishment of connection,  message from server: "Too many connections"
2019-10-14T01:13:34,631 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-14T01:15:44,033 [ERROR] (ConnectionPool.java:77) - java.sql.SQLNonTransientConnectionException: Data source rejected establishment of connection,  message from server: "Too many connections"
2019-10-14T01:15:44,061 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-14T01:27:48,529 [ERROR] (ConnectionPool.java:77) - java.sql.SQLNonTransientConnectionException: Data source rejected establishment of connection,  message from server: "Too many connections"
2019-10-14T01:27:48,546 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-14T01:29:02,519 [ERROR] (ConnectionPool.java:77) - java.sql.SQLNonTransientConnectionException: Data source rejected establishment of connection,  message from server: "Too many connections"
2019-10-14T01:29:02,535 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
2019-10-14T07:49:02,989 [ERROR] (ConnectionPool.java:77) - java.sql.SQLNonTransientConnectionException: Data source rejected establishment of connection,  message from server: "Too many connections"
2019-10-14T07:49:03,006 [ERROR] (ConnectionPool.java:62) - by.htp.pool.ConnectionPoolException: SQLExeption in ConnectionPool
